docs/subagent/gen_job.md

```
@@ -207,11 +205,32 @@ User: Migrate the users table from local_duckdb to greenplum,
+  5. Calls validate_ddl(database="greenplum", ddl=..., target_table="users_copy")
+     -> errors=[], proceeds
+  6. Executes DDL via execute_ddl(database="greenplum")
+  7. Calls transfer_query_result(source_database="local_duckdb",
+     target_database="greenplum", mode="replace")
+  8. Activates data-migration skill; runs 7 reconciliation checks
+  9. Returns migration report with pass/fail status
+```
+
+### Example 3: Migrate MySQL to StarRocks
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)